CorsixTH/Lua/app.lua

```diff
@@ -28,7 +28,7 @@ local runDebugger = corsixth.require("run_debugger")
 -- Increment each time a savegame break would occur
 -- and add compatibility code in afterLoad functions
 
-local SAVEGAME_VERSION = 159
+local SAVEGAME_VERSION = 160
```

Comment:
needs to bump again since hotkey was merged

Reply:
```suggestion
local SAVEGAME_VERSION = 161
```